web/frontend/src/components/base.vue

```diff
@@ -61,7 +62,8 @@ const keyboardStates = $ref({
   isActive: false,
 });
 
-const resources = $computed(() => filterResources(props.resources, 'rdk', 'component', 'camera'));
+const cameraResources = filterResources(props.resources, 'rdk', 'component', 'camera');
+const resources: Array<{name: string}> = $computed(() => cameraResources);
```

Comment:
```suggestion
const resources = $computed(() => filterResources(props.resources, 'rdk', 'component', 'camera'));
```

Reply:
Can do except the type is needed for resources or I get the following TS error.

```
[{
	"resource": "/Users/jason/code/rdk/web/frontend/src/components/base.vue",
	"owner": "_generated_diagnostic_collection_name_#0",
	"code": "2339",
	"severity": 8,
	"message": "Property 'name' does not exist on type 'never'.",
	"source": "ts",
	"startLineNumber": 200,
	"startColumn": 28,
	"endLineNumber": 200,
	"endColumn": 32
}]
```

Reply:
does this issue show up just as a vscode issue or also during `npm run typecheck`?

Reply:
Just in VS Code inline and Problems window

Reply:
Then I'd recommend removing `Array<{name: string}>` for now. We've got a bigger issue with `never` that we need to fix and this just masks it

Reply:
Sounds good

Reply:
thank you!